python/tvm/topi/transform.py

```diff
@@ -502,6 +502,14 @@ def tensordot(a, b, axes):
     return cpp.tensordot(a, b, axes[0], axes[1])
 
 
+def einsum(subscripts, a_tuple):
```

Comment:
doesn't seem right to be in transform.py

Reply:
Created einsum.py in topi for this op. Thx!